recipes/sqlite3/all/CMakeLists.txt

```diff
@@ -33,6 +34,8 @@ option(HAVE_POSIX_FALLOCATE "Use posix_fallocate()")
 option(HAVE_STRERROR_R "Use strerror_r()")
 option(HAVE_USLEEP "Use usleep() system call to implement the xSleep method")
 option(DISABLE_GETHOSTUUID "Disable function gethostuuid")
+option(MAX_COLUMN "The maximum number of columns in a table / index / view")
+option(MAX_VARIABLE_NUMBER "The maximum value of a ?nnn wildcard that the parser will accept")
```

Comment:
These 2 variables should be CACHE STRING variables, not CACHE BOOL (option() is CACHE BOOL).